addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -2850,692 +2850,6 @@ return Timers;
-     */
-    render: function (thread, options) {
-        var self = this;
-
-        var shouldScrollToBottomAfterRendering = false;
-        if (this._currentThreadID === thread.getID() && this.isAtBottom()) {
-            shouldScrollToBottomAfterRendering = true;
-        }
-        this._currentThreadID = thread.getID();
-
-        // copy so that reverse do not alter order in the thread object
-        var messages = _.clone(thread.getMessages({ domain: options.domain || [] }));
-
-        var modeOptions = options.isCreateMode ? this._disabledOptions :
-                                                    this._enabledOptions;
-
-        // attachments ordered by messages order (increasing ID)
-        this.attachments = _.uniq(_.flatten(_.map(messages, function (message) {
-            return message.getAttachments();
-        })));
-
-        options = _.extend({}, modeOptions, options, {
-            selectedMessageID: this._selectedMessageID,
-        });
-
-        // dict where key is message ID, and value is whether it should display
-        // the author of message or not visually
-        var displayAuthorMessages = {};
-
-        // Hide avatar and info of a message if that message and the previous
-        // one are both comments wrote by the same author at the same minute
-        // and in the same document (users can now post message in documents
-        // directly from a channel that follows it)
-        var prevMessage;
-        _.each(messages, function (message) {
-            if (
-                // is first message of thread
-                !prevMessage ||
-                // more than 1 min. elasped
-                (Math.abs(message.getDate().diff(prevMessage.getDate())) > 60000) ||
-                prevMessage.getType() !== 'comment' ||
-                message.getType() !== 'comment' ||
-                // from a different author
-                (prevMessage.getAuthorID() !== message.getAuthorID()) ||
-                (
-                    // messages are linked to a document thread
-                    (
-                        prevMessage.isLinkedToDocumentThread() &&
-                        message.isLinkedToDocumentThread()
-                    ) &&
-                    (
-                        // are from different documents
-                        prevMessage.getDocumentModel() !== message.getDocumentModel() ||
-                        prevMessage.getDocumentID() !== message.getDocumentID()
-                    )
-                )
-            ) {
-                displayAuthorMessages[message.getID()] = true;
-            } else {
-                displayAuthorMessages[message.getID()] = !options.squashCloseMessages;
-            }
-            prevMessage = message;
-        });
-
-        if (modeOptions.displayOrder === ORDER.DESC) {
-            messages.reverse();
-        }
-
-        this.$el.html(QWeb.render('im_livechat.legacy.mail.widget.Thread', {
```

Comment:
ci/security due to this move

Reply:
This one is fine, especially the value is coming from a template.